rules/aip0134/request_allow_missing_field_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0134
+
+import (
+	"testing"
+
+	"github.com/googleapis/api-linter/rules/internal/testutils"
+)
+
+func TestAllowMissing(t *testing.T) {
+	for _, test := range []struct {
+		name         string
+		Style        string
+		AllowMissing string
+		problems     testutils.Problems
+	}{
+		{"IgnoredNotDF", "", "", nil},
+		{"ValidIncluded", "style: DECLARATIVE_FRIENDLY", "bool allow_missing = 2;", nil},
+		{"Invalid", "style: DECLARATIVE_FRIENDLY", "", testutils.Problems{{Message: "include `bool allow_missing`"}}},
```

Comment:
Nit: maybe include an invalid type for `allow_missing` test? technically that condition is not covered by these tests.

```suggestion
		{"Invalid", "style: DECLARATIVE_FRIENDLY", "", testutils.Problems{{Message: "include `bool allow_missing`"}}},
		{"InvalidType", "style: DECLARATIVE_FRIENDLY", "string allow_missing = 2;", testutils.Problems{{Message: "include `bool allow_missing`"}}},
```

Reply:
Good point. Done.